### ISSUE TITLE
Stopping `rig project theme` with Control + C results in [ERROR]

### ISSUE THREAD

**Issue by anniegreens** [phase2/rig#93]
Since updating to rig 1.3.2 stopping a running theme with the usual Control + C does still stop, but also outputs the following error:

`^C[ERROR] Error running project script 'theme': 130`

Is there a more graceful way to stop this now or should this be expected?

**Comment by grayside** [comment]
I've yet to find any change in the rig code that could affect this. However, it is possible the underlying script doesn't gracefully handle the termination signal and throws an error (with a code 130 to be specific).

If you do this:
```
rig project run cli
npm install && npm run setup && npm start
# Cntrl C
echo $#
```

Do you get the 130 exit code? If so, we either need to live with it, or npm start can be fancied up.

Cc @illepic

**Comment by anniegreens** [comment]
@grayside I do not get the 130 exit code when running and ending it in that manner.

**Comment by anniegreens** [comment]
Aha but I do when I exit the cli:

` --------------------------------------------------------
[BS] Serving files from: ./
[BS] Watching files...
^C
[root@d1333e257096 ahs_theme]# exit
exit
[ERROR] Error running project script 'run': 130`

**Comment by grayside** [comment]
Per http://www.tldp.org/LDP/abs/html/exitcodes.html, it seems that code 130 is the official exit code for processes terminated with exit code 130, and other research I've done indicates that Docker containers (and presumably rig project run) will set this status.

Given this my initial thought is we should wontfix, but if we can intercept it may be friendlier to change the message to, say `[WARNING] User interrupted project script 'run': 130` to be a bit more clear. Does that seem reasonable?

**Comment by anniegreens** [comment]
👍 I think that's an acceptable compromise and a warning/notice seems more appropriate.

**Comment by grayside** [comment]
I'm having a hard time replicating this problem. In my environment if I hit Control C it immediately exits without a message. What shell are you using? Do you know if you have any special handling for signals?

**Comment by anniegreens** [comment]
Yeah, I am using zsh with some default OhMYZSH settings. I'd have to dig and see if there are any signal handling, as I'm unsure of that.
